src/third-party.md

```diff
@@ -36,6 +36,16 @@ Snap it into your existing workflows with the click of a button, automate away t
 * [The Last Pickle Cassandra stress tool, tlp-stress](https://github.com/thelastpickle/tlp-stress): A workload-centric stress tool for Apache Cassandra. Designed for simplicity, no math degree required. (DataStax)
 * [NoSQLBench](https://github.com/nosqlbench/nosqlbench): Pluggable benchmarking suite for Cassandra and other distributed systems
 * [Stargate](https://stargate.io): Open source data gateway providing Document, REST, and GraphQL APIs to Apache Cassandra.
+* [Instaclustr Esop](https://github.com/instaclustr/esop): Swiss knife for backup and restore of your node to GCP, Azure, S3, Ceph etc. Supports backup and restoration of commit logs too. Esop is embedded in [Instaclustr Icarus](https://github.com/instaclustr/icarus) sidecar so you may backup and restore your cluster remotely and on-the-fly with any disruption.
+* [Instaclustr Kerberos plugin](https://github.com/instaclustr/cassandra-kerberos): A GSSAPI authentication provider for Apache Cassandra
+* [Instaclustr Java Driver for Kerberos](https://github.com/instaclustr/cassandra-java-driver-kerberos): A GSSAPI authentication provider for the Cassandra Java driver.
+* [Instaclustr Minotaur](https://github.com/instaclustr/instaclustr-minotaur): Command line tool for consistent rebuilding of a Cassandra cluster.
+* [Instaclustr LDAP Authenticator](https://github.com/instaclustr/cassandra-ldap): LDAP Authenticator for Apache Cassandra
+* [Instaclustr TTL Remover](https://github.com/instaclustr/cassandra-ttl-remover): Command line tool for rewriting SSTables to remove TTLs
+* [Instaclustr SSTable Generator](https://github.com/instaclustr/cassandra-sstable-generator): CLI tool for programmatic generation of Cassandra SSTables
+* [Instaclustr Exporter](https://github.com/instaclustr/cassandra-exporter): Java agent that exports Cassandra metrics to Prometheus
+* [Instaclustr Go Client for Instaclustr Icarus](https://github.com/instaclustr/instaclustr-icarus-go-client): Go client for Instaclustr Icarus sidecar
+* [Instaclustr SStable Tools](https://github.com/instaclustr/cassandra-sstable-tools): A command line tool that helps admins get summaries, metadata, partition info, cell info for SSTables
```

Comment:
[Instaclustr SStable Tools](https://github.com/instaclustr/cassandra-sstable-tools): A command line tool that helps admins get summaries, metadata, partition info, and cell info for SSTables.